src/main/java/Activities.java

```diff
@@ -0,0 +1,27 @@
+abstract class Activities {
+    protected String Activity;
+    private int status = 0; // 0 represent undone: 1 represents done
+
+    public void doIt() {
+        status = 1;
+    }
+
+    public String getActivity(){
+        return Activity;
+    }
+    public int isDone(){
```

Comment:
The method isDone() sounds like having the return type as boolean. How about changing the integer value of status to boolean?